docs/document_loader.ipynb

```diff
@@ -4,72 +4,120 @@
+     ]
+    }
+   ],
+   "source": [
+    "docs_iterator = loader.lazy_load()\n",
+    "for doc in docs_iterator:\n",
+    "    print(doc)\n",
+    "    break"
```

Comment:
nit: should probably just be: `for doc in loader.lazy_load()`

Reply:
Done